src/applications/gqrx/mainwindow.cpp

```diff
@@ -464,6 +467,10 @@ bool MainWindow::loadConfig(const QString cfgfile, bool check_crash,
         restoreState(m_settings->value("gui/state", saveState()).toByteArray());
     }
 
+    // Restore settings menu hide/show state
+    bool_val = m_settings->value("gui/showmenu", false).toBool();
```

Comment:
"showmenu" is a misleading name since these widgets are control panels and now menus. Furthermore, the default value should be true (visible) since that's how the program works now.